README.md

```diff
@@ -84,6 +84,59 @@ From the root dir run:
 sudo env PYTHONPATH=. python <file>
 ```
 
+### Docker Installation and Usage
+
+#### Pre-requisites
+
+To install the **project's Docker installation pre-requisites**, please follow the instructions in the link below:
+
+- [Docker](https://docs.docker.com/get-docker/)
+
+> _**Note**: if you're using a Linux system, please take a look at [Docker's post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)!_
+
+#### Building and running
+
+Once you have all pre-requisites installed, change your current working directory to the project's root:
+
+/```bash
```

Comment:
I believe you forgot to remove the `/` :joy: 
I added the `\` because GitHub bugs the preview if in certain cases :stuck_out_tongue: 

```suggestion
```bash
```

Reply:
Done